src/main/java/com/indore/api/UserprofileSearchResult.java

```diff
@@ -0,0 +1,143 @@
+package com.indore.api;
+
+/**
+ *   UserprofileSearchResult to generate getters for UserProfile Resource.
+ *   getter is a method that reads value of a variable.
+ */
+
+public final class UserprofileSearchResult {
```

Comment:
As discussed, User is the single entity, hence you need to enhance existing `UserSearchResult` class to include other properties of user from `UserProfile` class and use builder pattern in `UserSearchResult`.